applications/feedback_sentiment_analyzer/cdk/lib/functions.ts

```diff
@@ -3,11 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import {resolve} from "path";
-import {BundlingOutput, Duration} from "aws-cdk-lib";
-import {Code, Runtime} from "aws-cdk-lib/aws-lambda";
+import { resolve } from "path";
```

Comment:
@scmacdon could you undo these formatting changes? We have other linters that prefer no whitespace between curly braces. Trying to be consistent here.